File: URI-Judge-Online-python-language/1181.py
a = [[0 for y in range(12)] for x in range(12)]
letra = str(input())
soma=0
for x in range(12):
    for y in range(12):
        a[x][y]=float(input())
for x in range(12):
    for y in range(12):
        if y>x:
            soma+=a[x][y]

if(letra=='S'):
    print("{:.1f}".format(soma))
if(letra=='M'):
    print("{:.1f}".format(soma/66))

File: URI-Online-Judge-c/1181.c
#include <stdio.h>

int main(){
    float soma=0;
    int linha;
    int i=0;
    int j=0;
    char letra;
    float a[2][2];
    scanf("%d %c",&linha,&letra);
    for(i=0;i<2;i++){
        for(j=0;j<2;j++){
            scanf("%f",&a[i][j]);
        }

    }
    for(j=0;j<2;j++){
         soma+=a[linha][j];
    }
        if (letra=='S'){
            printf("%f",soma);
        }
         else if(letra=='M'){
            printf("%f",soma/2);
            }


    return 0;
}
